### PR TITLE
Do-310: Fixed the issue with website URL.

### PR DESCRIPTION
**Pull request by dhruvlahoti**

### CODE REVIEW

File: src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx [hunk 1]
```diff
@@ -58,6 +61,13 @@ const UrlField = (props) => {
     let value = event ? event.target.value : '';
     if (isEmpty(value)) {
       onChange();
+    } else if (unsecured) {
+      if (value.includes('https://') || value.includes('http://')) {
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx:65]: Simplify this condition 
```suggestion
      if (!value.includes('https://') || !value.includes('http://')) {
        value = `http://${value}`;
      }
```

File: src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx [hunk 2]
```diff
@@ -58,6 +61,13 @@ const UrlField = (props) => {
...
+        // Do nothing
+      } else {
+        value = `http://${value}`;
+      }
+      onChange(value);
     } else {
       if (value.includes('https://')) {
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx:72]: Similarly we can simplify this condition as well.

File: src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx (hunk 1 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx]: Done

File: src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx (hunk 2 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx]: Done

File: src/utils/common/formUtils.js [hunk 3]
```diff
@@ -187,9 +190,14 @@ export const createFormUISchema = ({ formData, excludeField, fieldTypeMap }) =>
               uiField = {
                 'ui:field': 'DatePickerInput',
               };
+            } else if (urlSecuredFields.includes(field)) {
+              uiField = {
+                'ui:field': 'UrlField',
```
Comment (pepcus-ankit) [inline src/utils/common/formUtils.js:195]: why do we need both urlSecuredFields & urlFields if we have a unsecured property. 

I think we should have urlFields in that isSecured property (with value as true/false with default as true)

Comment (dhruvlahoti) [reply src/utils/common/formUtils.js:195]: We are doing the same thing, urlSecuredFields & urlFields are the arrays of the fields, the secured fields will be in urlSecuredFields arrays and vice versa

Comment (dhruvlahoti) [reply src/utils/common/formUtils.js]: Done

File: src/utils/common/formUtils.js [hunk 4]
```diff
@@ -187,10 +223,8 @@ export const createFormUISchema = ({ formData, excludeField, fieldTypeMap }) =>
               uiField = {
                 'ui:field': 'DatePickerInput',
               };
-            } else if (urlFields.includes(field)) {
-              uiField = {
-                'ui:field': 'UrlField',
-              };
+            } else if (!isEmpty(getUrlField(field))) {
```
Comment (mohitsaladiya) [inline src/utils/common/formUtils.js:226]: We should hold the method response in a const. So that we don't have to call it twice.

Comment (dhruvlahoti) [reply src/utils/common/formUtils.js]: Done

**Pull request merged**